test/tests/functional/pbs_reliable_job_startup.py

```diff
@@ -5452,6 +5452,8 @@ def test_t20(self):
                 sjid, iexec_vnode), n=10)
             self.momA.log_match("Job;%s;pruned to exec_vnode=%s" % (
                 sjid, exec_vnode), n=10)
+
+        for idx in range(1, 4):
```

Comment:
Just attaching the outer for loop here would just result in checking accounting_logs for sjid_esc, iexec_host_esc, and iexec_vnode_esc values of the subjob at index 3 from previous for loop, and running that 3 times.
The sjid_esc, iexec_host_esc, and iexec_vnode_esc values from previous loop must be saved as arrays indexed by the subjob index. Then the individual subjob entries can be checked as sjid_esc[idx], iexec_host_esc[idx], and iexec_vnode_esc[idx]...